mmgen/models/architectures/lsgan/generator_discriminator.py

```diff
@@ -0,0 +1,297 @@
+import math
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule
+from mmcv.cnn.bricks import build_activation_layer
+
+from mmgen.models.builder import MODULES
+from ..common import get_module_device
+
+
+@MODULES.register_module()
+class LSGANGenerator(nn.Module):
+    """Generator for LSGAN.
+
+    Implementation Details for LSGAN architecture:
+
+    #. Adopt transposed convolution in the generator;
```

Comment:
These descriptions are for DCGAN. Please detail the implementation for LSGAN Generator. (e.g. difference from DCGAN)